packages/typespec-azure-resource-manager/lib/operations.tsp

```diff
@@ -694,3 +694,140 @@ op checkLocalNameAvailability<
   Response,
   AdditionalParams
 >;
+
+/**
+ * @dev A provider action performed over a subscription
+ * @template Request The request model for the action.
+ * @template Response The response type for the action.
+ * @template Parameters Optional. Additional parameters after the path parameters (e.g. Location)
+ * @template Scope Optional. The scope of the action (defaults to SubscriptionActionScope).
+ * @template Error Optional. The error response, if non-standard.
+ */
+op ArmProviderActionSync<
+  Request extends {} | void = void,
+  Response extends {} | void = void,
+  Parameters extends {} = {},
+  Scope extends SubscriptionActionScope = SubscriptionActionScope,
+  Error extends {} = ErrorResponse
+> is ArmActionSync<Request, Response, Parameters, Scope, Error>;
+
+/**
+ * @dev A long-running provider action performed over a subscription.
+ * @template Request The request model for the action
+ * @template Response The response type for the action
+ * @template Parameters Optional. Additional parameters after the path parameters (e.g. Location)
+ * @template Scope Optional. The scope of the action (defaults to SubscriptionActionScope).
+ * @template LroHeaders Optional.  Allows overriding the lro headers returned in the Accepted response
+ * @template Error Optional. The error response, if non-standard.
+ */
+op ArmProviderActionAsync<
+  Request extends {} | void = void,
+  Response extends {} | void = void,
+  Parameters extends {} = {},
+  Scope extends SubscriptionActionScope = SubscriptionActionScope,
+  LroHeaders extends TypeSpec.Reflection.Model = ArmLroLocationHeader<
+    Azure.Core.StatusMonitorPollingOptions<ArmOperationStatus>,
+    Response,
+    string
+  > &
+    Azure.Core.Foundations.RetryAfterHeader,
+  Error extends {} = ErrorResponse
+> is ArmActionAsync<Request, Response, Parameters, Scope, LroHeaders, Error>;
+
+/**
+ * @dev A provider action performed over a tenant
+ * @template Request The request model for the action
+ * @template Response The response type for the action
+ * @template Parameters Optional. Additional parameters after the path parameters (e.g. Location)
+ * @template Scope Optional. The scope of the action (Defaults to TenantActionScope).
+ * @template Error Optional. The error response, if non-standard.
+ */
+op ArmTenantActionSync<
```

Comment:
I don't think we need `ArmTenantActionXXX` and `ArmActionXXX` as Scope can be used to adjust the scope path. Please update the `@param Scope` comment to indicate the current two choices.

Reply:
sure! done :)